### PR TITLE
Add Benchmark results to remote store main page

### PR DESCRIPTION
**Pull request by Naarcha-AWS**
Fixes issue #5052.

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by ashking94 — APPROVED** [review]

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md [hunk 1]
```diff
@@ -104,9 +104,65 @@ If the Security plugin is enabled, a user must have the `cluster:admin/remotesto
...
+
+* Node types: Three nodes, one data, one ingest, and one cluster manager node
+* Node instance: AWS EC2 r6g.xlarge
+* OpenSearch Benchmark host: Single AWS EC2 m5.2xlarge
+* Shard configuration: 3 shards with 1 replica
+* The `repository-s3` plugin installed with the default S3 settings 
+
+### StackOverflow
+
+The following table lists the benchmarking results for the `so` workload a remote translog buffer interval of 250ms.
```
Comment (Naarcha-AWS) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:134]:
```suggestion
The following table lists the benchmarking results for the `so` workload a remote translog buffer interval of 250 ms.
```

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md [hunk 2]
```diff
@@ -104,9 +104,65 @@ If the Security plugin is enabled, a user must have the `cluster:admin/remotesto
...
+|	|	|8 bulk indexing clients (Default)	|16 bulk indexing clients	|24 bulk indexing clients	|
+|---	|---	|---	|---	|---	|
+|	|	|Document Replication	|remote enabled	|Percent difference	|Document Replication	|remote enabled	|Percent difference	|Document Replication	|remote enabled	|Percent difference	|
+|Indexing Throughput	|Mean	|29582.5	|40667.4	|37.47	|31154.9	|47862.3	|53.63	|31777.2	|51123.2	|60.88	|
+|P50	|28915.4	|40343.4	|39.52	|30406.4	|47472.5	|56.13	|30852.1	|50547.2	|63.84	|
+|Indexing Latency	|P90	|1716.34	|1469.5	|-14.38	|3709.77	|2799.82	|-24.53	|5768.68	|3794.13	|-34.23	|
+
+### HTTP logs
+
+The following table lists the benchmarking results for the `http_logs` workload with a remote translog buffer interval of 200ms.
```
Comment (Naarcha-AWS) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:145]:
```suggestion
The following table lists the benchmarking results for the `http_logs` workload with a remote translog buffer interval of 200 ms.
```

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md [hunk 3]
```diff
@@ -104,9 +104,65 @@ If the Security plugin is enabled, a user must have the `cluster:admin/remotesto
...
+|	|	|8 bulk indexing clients (Default)	|16 bulk indexing clients	|24 bulk indexing clients	|
+|---	|---	|---	|---	|---	|
+|	|	|Document Replication	|remote enabled	|Percent difference	|Document Replication	|remote enabled	|Percent difference	|Document Replication	|remote enabled	|Percent difference	|
+|Indexing Throughput	|Mean	|149062	|82198.7	|-44.86	|134696	|148749	|10.43	|133050	|197239	|48.24	|
+|P50	|148123	|81656.1	|-44.87	|133591	|148859	|11.43	|132872	|197455	|48.61	|
+|Indexing Latency	|P90	|327.011	|610.036	|86.55	|751.705	|669.073	|-10.99	|1145.19	|817.185	|-28.64	|
+
+### NYC taxis
+
+The following table lists the benchmarking results for the `http_logs` workload with a remote translog buffer interval of 250ms.
```
Comment (Naarcha-AWS) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:156]:
```suggestion
The following table lists the benchmarking results for the `http_logs` workload with a remote translog buffer interval of 250 ms.
```

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md [hunk 4]
```diff
@@ -104,9 +104,65 @@ If the Security plugin is enabled, a user must have the `cluster:admin/remotesto
...
+
+|	|	|8 bulk indexing clients (Default)	|16 bulk indexing clients	|24 bulk indexing clients	|
+|---	|---	|---	|---	|---	|
+|	|	|Document Replication	|remote enabled	|Percent difference	|Document Replication	|remote enabled	|Percent difference	|Document Replication	|remote enabled	|Percent difference	|
+|Indexing Throughput	|Mean	|93383.9	|94186.1	|0.86	|91624.8	|125770	|37.27	|93627.7	|132006	|40.99	|
+|P50	|91645.1	|93906.7	|2.47	|89659.8	|125443	|39.91	|91120.3	|132166	|45.05	|
+|Indexing Latency	|P90	|995.217	|1014.01	|1.89	|2236.33	|1750.06	|-21.74	|3353.45	|2472	|-26.28	|
+
+As detailed by the results, you can see consistent gains for cases where the indexing latency is more than the average remote upload time. When you increase the bulk indexing clients, remote-enabled configuration provides indexing throughput gains of up to 60-65%. For more detailed results, see [Issue #9790](https://github.com/opensearch-project/OpenSearch/issues/9790).
```
Comment (Naarcha-AWS) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:165]:
```suggestion
As detailed by the results, you can see consistent gains for cases where the indexing latency is more than the average remote upload time. When you increase the bulk indexing clients, remote-enabled configuration provides indexing throughput gains of up to 60--65%. For more detailed results, see [Issue #9790](https://github.com/opensearch-project/OpenSearch/issues/9790).
```

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md [hunk 5]
```diff
@@ -104,9 +104,65 @@ If the Security plugin is enabled, a user must have the `cluster:admin/remotesto
...
 You can use remote-backed storage for the following purposes:
 
-- To restore red clusters or indexes
-- To recover all data up to the last acknowledged write, regardless of replica count, if `index.translog.durability` is set to `request`
+- To restore red clusters or indexes.
+- To recover all data up to the last acknowledged write, regardless of replica count, if `index.translog.durability` is set to `request`.
 
+## Benchmarks
+
+The OpenSearch team has run remote store using multiple workload options available within [OpenSearch Benchmark (OSB)](https://opensearch.org/docs/latest/benchmark/index/) tool. This section summarises the benchmark results for the following OSB workloads:
```
Comment (cwillum) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:112]:
```suggestion
The OpenSearch team has run remote store using multiple workload options available within the [OpenSearch Benchmark (OSB)](https://opensearch.org/docs/latest/benchmark/index/) tool. This section summarizes the benchmark results for the following OSB workloads: 
```

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md [hunk 6]
```diff
@@ -104,9 +104,65 @@ If the Security plugin is enabled, a user must have the `cluster:admin/remotesto
...
 
+## Benchmarks
+
+The OpenSearch team has run remote store using multiple workload options available within [OpenSearch Benchmark (OSB)](https://opensearch.org/docs/latest/benchmark/index/) tool. This section summarises the benchmark results for the following OSB workloads: 
+
+- [StackOverflow](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/so)
+- [HTTP logs](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/http_logs)
+- [NYC taxis](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/nyc_taxis),
+
+Each workload was tested using multiple cluster configurations indexing data using bulk indexing clients, which simulates a high level of requests made to the cluster concurrently.
```
Comment (cwillum) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:118]: Not sure what this means. Reword this?

"... using multiple cluster configurations indexing data using bulk indexing clients ..."

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md [hunk 7]
```diff
@@ -104,9 +104,65 @@ If the Security plugin is enabled, a user must have the `cluster:admin/remotesto
...
+
+* Node types: Three nodes, one data, one ingest, and one cluster manager node
+* Node instance: AWS EC2 r6g.xlarge
+* OpenSearch Benchmark host: Single AWS EC2 m5.2xlarge
+* Shard configuration: 3 shards with 1 replica
+* The `repository-s3` plugin installed with the default S3 settings 
+
+### StackOverflow
+
+The following table lists the benchmarking results for the `so` workload a remote translog buffer interval of 250 ms.
```
Comment (cwillum) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:134]:
```suggestion
The following table lists the benchmarking results for the `so` workload with a remote translog buffer interval of 250 ms.
```

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md (hunk 6 above)
Comment (Naarcha-AWS) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:118]:
```suggestion
Each workload was tested using multiple cluster configurations. Indexed data was added to each cluster using varying numbers of bulk indexing clients. This simulates a high level of requests made to the cluster concurrently. 
```

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md [hunk 8]
```diff
@@ -104,9 +104,65 @@ If the Security plugin is enabled, a user must have the `cluster:admin/remotesto
...
 
+## Benchmarks
+
+The OpenSearch team has run remote store using multiple workload options available within the [OpenSearch Benchmark (OSB)](https://opensearch.org/docs/latest/benchmark/index/) tool. This section summarizes the benchmark results for the following OSB workloads: 
+
+- [StackOverflow](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/so)
+- [HTTP logs](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/http_logs)
+- [NYC taxis](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/nyc_taxis),
+
+Each workload was tested using multiple cluster configurations. Indexed data was added to each cluster using varying numbers of bulk indexing clients. This simulates a high level of requests made to the cluster concurrently.
```
Comment (ashking94) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:118]: This line conveys a different meaning. Can we reword this as - `Each workload was tested against multiple configurations of bulk indexing clients which simulates varying degree of concurrency of requests.`

**Review by ashking94 — CHANGES_REQUESTED** [review]
Please make the suggested change.

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md [hunk 9]
```diff
@@ -104,9 +104,65 @@ If the Security plugin is enabled, a user must have the `cluster:admin/remotesto
...
+* Node instance: AWS EC2 r6g.xlarge
+* OpenSearch Benchmark host: Single AWS EC2 m5.2xlarge
+* Shard configuration: 3 shards with 1 replica
+* The `repository-s3` plugin installed with the default S3 settings 
+
+### StackOverflow
+
+The following table lists the benchmarking results for the `so` workload with a remote translog buffer interval of 250 ms.
+
+|	|	|8 bulk indexing clients (Default)	|16 bulk indexing clients	|24 bulk indexing clients	|
```
Comment (natebower) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:136]: Confirm sentence case for the items in this and the following tables.

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md [hunk 10]
```diff
@@ -104,9 +104,65 @@ If the Security plugin is enabled, a user must have the `cluster:admin/remotesto
...
+The following table lists the benchmarking results for the `http_logs` workload with a remote translog buffer interval of 250 ms.
+
+|	|	|8 bulk indexing clients (Default)	|16 bulk indexing clients	|24 bulk indexing clients	|
+|---	|---	|---	|---	|---	|
+|	|	|Document Replication	|remote enabled	|Percent difference	|Document Replication	|remote enabled	|Percent difference	|Document Replication	|remote enabled	|Percent difference	|
+|Indexing Throughput	|Mean	|93383.9	|94186.1	|0.86	|91624.8	|125770	|37.27	|93627.7	|132006	|40.99	|
+|P50	|91645.1	|93906.7	|2.47	|89659.8	|125443	|39.91	|91120.3	|132166	|45.05	|
+|Indexing Latency	|P90	|995.217	|1014.01	|1.89	|2236.33	|1750.06	|-21.74	|3353.45	|2472	|-26.28	|
+
+As detailed by the results, you can see consistent gains for cases where the indexing latency is more than the average remote upload time. When you increase the bulk indexing clients, remote-enabled configuration provides indexing throughput gains of up to 60--65%. For more detailed results, see [Issue #9790](https://github.com/opensearch-project/OpenSearch/issues/9790).
```
Comment (natebower) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:165]:
```suggestion
As shown by the results, there are consistent gains in cases where the indexing latency is more than the average remote upload time. When you increase the bulk indexing clients, remote-enabled configuration provides indexing throughput gains of up to 60--65%. For more detailed results, see [Issue #9790](https://github.com/opensearch-project/OpenSearch/issues/9790).
```

Comment (natebower) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:165]: Second sentence: Do we mean "When the number of bulk indexing clients is increased..."? Should "a" precede "remote-enabled"?

**Review by natebower — CHANGES_REQUESTED** [review]
@Naarcha-AWS I'd like verify some of the changes on this one, so please tag me once you've addressed my comments. Thanks!

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md (hunk 8 above)
Comment (Naarcha-AWS) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:118]:
```suggestion
Each workload was tested against multiple bulk indexing client configurations in order to simulate varying degrees of request concurrency.
```

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md (hunk 10 above)
Comment (Naarcha-AWS) [reply _tuning-your-cluster/availability-and-recovery/remote-store/index.md:165]: Yes and yes.

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md [hunk 11]
```diff
@@ -102,11 +102,67 @@ If the Security plugin is enabled, a user must have the `cluster:admin/remotesto
...
+|P50	|148123	|81656.1	|-44.87	|133591	|148859	|11.43	|132872	|197455	|48.61	|
+|Indexing Latency	|P90	|327.011	|610.036	|86.55	|751.705	|669.073	|-10.99	|1145.19	|817.185	|-28.64	|
+
+### NYC taxis
+
+The following table lists the benchmarking results for the `http_logs` workload with a remote translog buffer interval of 250 ms.
+
+|	|	|8 bulk indexing clients (Default)	|16 bulk indexing clients	|24 bulk indexing clients	|
+|---	|---	|---	|---	|---	|
+|	|	|Document Replication	|remote enabled	|Percent difference	|Document Replication	|remote enabled	|Percent difference	|Document Replication	|remote enabled	|Percent difference	|
```
Comment (Naarcha-AWS) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:160]:
```suggestion
|	|	| Document replication	| Remote enabled	|Percent difference	| Document replication	| Remote enabled	| Percent difference	|Document replication	| Remote enabled	| Percent difference	|
```

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md [hunk 12]
```diff
@@ -102,11 +102,67 @@ If the Security plugin is enabled, a user must have the `cluster:admin/remotesto
...
+|P50	|28915.4	|40343.4	|39.52	|30406.4	|47472.5	|56.13	|30852.1	|50547.2	|63.84	|
+|Indexing Latency	|P90	|1716.34	|1469.5	|-14.38	|3709.77	|2799.82	|-24.53	|5768.68	|3794.13	|-34.23	|
+
+### HTTP logs
+
+The following table lists the benchmarking results for the `http_logs` workload with a remote translog buffer interval of 200 ms.
+
+|	|	|8 bulk indexing clients (Default)	|16 bulk indexing clients	|24 bulk indexing clients	|
+|---	|---	|---	|---	|---	|
+|	|	|Document Replication	|remote enabled	|Percent difference	|Document Replication	|remote enabled	|Percent difference	|Document Replication	|remote enabled	|Percent difference	|
```
Comment (Naarcha-AWS) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:149]:
```suggestion
|	|	|Document Replication	|remote enabled	|Percent difference	|Document Replication	|remote enabled	|Percent difference	|Document Replication	|remote enabled	|Percent difference	|
```

Comment (Naarcha-AWS) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:149]:
```suggestion
|	|	| Document replication	| Remote enabled	|Percent difference	| Document replication	| Remote enabled	| Percent difference	|Document replication	| Remote enabled	| Percent difference	|
```

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md [hunk 13]
```diff
@@ -102,11 +102,67 @@ If the Security plugin is enabled, a user must have the `cluster:admin/remotesto
...
+* Shard configuration: Three shards with one replica
+* The `repository-s3` plugin installed with the default S3 settings 
+
+### StackOverflow
+
+The following table lists the benchmarking results for the `so` workload with a remote translog buffer interval of 250 ms.
+
+|	|	|8 bulk indexing clients (Default)	|16 bulk indexing clients	|24 bulk indexing clients	|
+|---	|---	|---	|---	|---	|
+|	|	|Document Replication	|remote enabled	|Percent difference	|Document Replication	|remote enabled	|Percent difference	|Document Replication	|remote enabled	|Percent difference	|
```
Comment (Naarcha-AWS) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:138]:
```suggestion
|	|	| Document replication	| Remote enabled	|Percent difference	| Document replication	| Remote enabled	| Percent difference	|Document replication	| Remote enabled	| Percent difference	|
```

**Review by ashking94 — APPROVED** [review]
LGTM

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md [hunk 14]
```diff
@@ -102,11 +102,67 @@ If the Security plugin is enabled, a user must have the `cluster:admin/remotesto
...
+* The `repository-s3` plugin installed with the default S3 settings 
+
+### StackOverflow
+
+The following table lists the benchmarking results for the `so` workload with a remote translog buffer interval of 250 ms.
+
+|	|	|8 bulk indexing clients (Default)	|16 bulk indexing clients	|24 bulk indexing clients	|
+|---	|---	|---	|---	|---	|
+|	|	| Document replication	| Remote enabled	|Percent difference	| Document replication	| Remote enabled	| Percent difference	|Document replication	| Remote enabled	| Percent difference	|
+|Indexing Throughput	|Mean	|29582.5	|40667.4	|37.47	|31154.9	|47862.3	|53.63	|31777.2	|51123.2	|60.88	|
```
Comment (Naarcha-AWS) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:139]:
```suggestion
|Indexing throughput	|Mean	|29582.5	|40667.4	|37.47	|31154.9	|47862.3	|53.63	|31777.2	|51123.2	|60.88	|
```

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md [hunk 15]
```diff
@@ -102,11 +102,67 @@ If the Security plugin is enabled, a user must have the `cluster:admin/remotesto
...
+|P50	|28915.4	|40343.4	|39.52	|30406.4	|47472.5	|56.13	|30852.1	|50547.2	|63.84	|
+|Indexing Latency	|P90	|1716.34	|1469.5	|-14.38	|3709.77	|2799.82	|-24.53	|5768.68	|3794.13	|-34.23	|
```
Comment (Naarcha-AWS) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:141]:
```suggestion
|Indexing latency	|P90	|1716.34	|1469.5	|-14.38	|3709.77	|2799.82	|-24.53	|5768.68	|3794.13	|-34.23	|
```

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md [hunk 16]
```diff
@@ -102,11 +102,67 @@ If the Security plugin is enabled, a user must have the `cluster:admin/remotesto
...
+
+### HTTP logs
+
+The following table lists the benchmarking results for the `http_logs` workload with a remote translog buffer interval of 200 ms.
+
+|	|	|8 bulk indexing clients (Default)	|16 bulk indexing clients	|24 bulk indexing clients	|
+|---	|---	|---	|---	|---	|
+|	|	| Document replication	| Remote enabled	|Percent difference	| Document replication	| Remote enabled	| Percent difference	|Document replication	| Remote enabled	| Percent difference	|
+|Indexing Throughput	|Mean	|149062	|82198.7	|-44.86	|134696	|148749	|10.43	|133050	|197239	|48.24	|
```
Comment (Naarcha-AWS) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:150]:
```suggestion
|Indexing throughput	|Mean	|149062	|82198.7	|-44.86	|134696	|148749	|10.43	|133050	|197239	|48.24	|
```

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md [hunk 17]
```diff
@@ -102,11 +102,67 @@ If the Security plugin is enabled, a user must have the `cluster:admin/remotesto
...
+|P50	|148123	|81656.1	|-44.87	|133591	|148859	|11.43	|132872	|197455	|48.61	|
+|Indexing Latency	|P90	|327.011	|610.036	|86.55	|751.705	|669.073	|-10.99	|1145.19	|817.185	|-28.64	|
```
Comment (Naarcha-AWS) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:152]:
```suggestion
|Indexing latency	|P90	|327.011	|610.036	|86.55	|751.705	|669.073	|-10.99	|1145.19	|817.185	|-28.64	|
```

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md [hunk 18]
```diff
@@ -102,11 +102,67 @@ If the Security plugin is enabled, a user must have the `cluster:admin/remotesto
...
+
+### NYC taxis
+
+The following table lists the benchmarking results for the `http_logs` workload with a remote translog buffer interval of 250 ms.
+
+|	|	|8 bulk indexing clients (Default)	|16 bulk indexing clients	|24 bulk indexing clients	|
+|---	|---	|---	|---	|---	|
+|	|	| Document replication	| Remote enabled	|Percent difference	| Document replication	| Remote enabled	| Percent difference	|Document replication	| Remote enabled	| Percent difference	|
+|Indexing Throughput	|Mean	|93383.9	|94186.1	|0.86	|91624.8	|125770	|37.27	|93627.7	|132006	|40.99	|
```
Comment (Naarcha-AWS) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:161]:
```suggestion
|Indexing throughput	|Mean	|93383.9	|94186.1	|0.86	|91624.8	|125770	|37.27	|93627.7	|132006	|40.99	|
```

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md [hunk 19]
```diff
@@ -102,11 +102,67 @@ If the Security plugin is enabled, a user must have the `cluster:admin/remotesto
...
+|P50	|91645.1	|93906.7	|2.47	|89659.8	|125443	|39.91	|91120.3	|132166	|45.05	|
+|Indexing Latency	|P90	|995.217	|1014.01	|1.89	|2236.33	|1750.06	|-21.74	|3353.45	|2472	|-26.28	|
```
Comment (Naarcha-AWS) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:163]:
```suggestion
|Indexing latency	|P90	|995.217	|1014.01	|1.89	|2236.33	|1750.06	|-21.74	|3353.45	|2472	|-26.28	|
```

**Review by natebower — APPROVED** [review]
LGTM
